mkosi

```diff
@@ -117,6 +118,8 @@ GPT_ROOT_ARM_VERITY    = uuid.UUID("7386cdf2203c47a9a498f2ecce45a2d6")
 GPT_ROOT_ARM_64_VERITY = uuid.UUID("df3300ced69f4c92978c9bfb0f38d820")
 GPT_ROOT_IA64_VERITY   = uuid.UUID("86ed10d5b60745bb8957d350f23d0571")
 
+BIOS_PARTITION_SIZE = 1024 * 1024
```

Comment:
this should probably be configurable like the other sizes...

Reply:
Not sure about that... This is just a workaround for not having enough space to fit grub in  (the workaround for MBR partition is to leave a gap before the first partition, which is not possible for GPT). The use case is different than e.g. ESP in which you are going to put your kernels there.

Reply:
could you add a comment about that to the definition?

Reply:
sure